builder/package.json

```diff
@@ -3,14 +3,19 @@
     "version": "1.0.0",
     "description": "",
     "dependencies": {
+        "@fortawesome/fontawesome-free": "^5.15.2",
         "bootstrap": "^4.3.1",
+        "github-markdown-css": "^4.0.0",
```

Comment:
No version was included in the static version of `github-markdown-css` so it's unclear if this is compatible